src/main/java/com/artipie/api/ApiRepoCreateSlice.java

```diff
@@ -86,13 +86,13 @@ public Response response(final String line,
         // @checkstyle LineLengthCheck (50 lines)
         // @checkstyle ReturnCountCheck (50 lines)
         return new AsyncResponse(
-            Single.fromCallable(this.settings::storage).map(RxStorageWrapper::new).flatMap(
-                storage -> storage.exists(key).flatMap(
+            Single.fromCallable(() -> this.storage).map(RxStorageWrapper::new).flatMap(
```

Comment:
@olenagerasimova do not really understand why it is needed to keep this `Single.fromCallable(...)` part, it could be simplified to `new RxStorageWrapper(this.storage).exists().map(...)`